wso2cep/4.0.0/samples/cartridges/ec2/wso2cep-400/wso2cep-400-presenter.json

```diff
@@ -70,6 +70,10 @@
             "value": "root"
         },
         {
+            "name": "payload_parameter.CONFIG_PARAM_REGISTRY_DB_DRIVER",
+            "value": "com.mysql.jdbc.Driver"
+        },
+        {
```

Comment:
We can define driver in the module.ini as this is the default driver we use. If we want we can use the cartridge to override that value. So better to remove these from the cartridge.